doc/source/rllib/rllib-algorithms.rst

```diff
@@ -217,13 +218,13 @@ Asynchronous Proximal Policy Optimization (APPO)
     **APPO architecture:** APPO is an asynchronous variant of :ref:`Proximal Policy Optimization (PPO) <ppo>` based on the IMPALA architecture,
     but using a surrogate policy loss with clipping, allowing for multiple SGD passes per collected train batch.
     In a training iteration, APPO requests samples from all EnvRunners asynchronously and the collected episode
-    samples are returned to the main algo process as ray references (rather than actual objects available on the local algo process).
-    These episode references are then passed to the Learner(s) for asynchronous updates of the model(s).
-    To account for the fact that this asynchronous design leads to some degree of off-policiness
-    on the EnvRunners (models are not always synched back to EnvRunners right after a new version of the weights is available), APPO uses
-    a procedure called v-trace, `described in the IMPALA paper here <https://arxiv.org/abs/1802.01561>`__.
+    samples are returned to the main algorithm process as Ray references rather than actual objects available on the local process.
+    APPO then passes these episode references to the Learners for asynchronous updates of the model.
+    RLlib doesn't always synch back the weights to the EnvRunners right after a new model version is available.
+    To account for the resulting off-policy'ness on the EnvRunners, APPO uses a procedure called v-trace,
```

Comment:
```suggestion
    To account for the EnvRunners being off-policy, APPO uses a procedure called v-trace,
```

Reply:
Just trying to get around saying "off-policy'ness". Please correct as needed.

Reply:
Ok, makes sense. I'll review ...